src/_Concert.scss

```diff
@@ -0,0 +1,75 @@
+
+
+* {
+  box-sizing: border-box;
+}
+
+.concert { 
+  overflow: hidden;
+  height: 200px;
+  width: 250px;
+  color: black;
+  border-radius: 3%;
+/*  background-color: #fff;
+  border: 1px solid black;
+*/  box-shadow: 0 2px 2px rgb(0, 0, 0);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  margin: 0px auto;
+  cursor: pointer;
+}
+
+.concert-image {
+  width: 160%;
+  left: -85px;
+  top: -5px;
+  /*height: 80%;*/
+  margin: 5px 5px 0px 5px;
+  overflow: hidden;
+  position: absolute;
+}
+.concert-date,
+.concert-location {
+  font-size: .8em;
+}
+.concert-text-display {
+  height: 80px;
+  background: #3a3a3a;
+  opacity: .9;
+  z-index: 2;
+  margin-top: 115px;
+  width: 100%;
+  color: #fafafa;
+  padding: 5px auto auto 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.concert:hover .fa-play-circle {
+  opacity: 1;
+  transition: .3s;
+  font-size: 2.6rem;
+}
+
+p {
```

Comment:
Could be a good spot to do some nesting within the text container rule, just so that we're sure we don't accidentally affect any p tags that might be in other parts of the page/ main compiled CSS file.

Reply:
I also like nesting for hover states cause it keeps them nicely grouped together with the element you're hovering over/adding the hover state to.